### ISSUE TITLE
Contiribute documentation referes to makefile targets that don't exist

### ISSUE THREAD

**Issue by nemith** [nornir-automation/nornir#862]
I was reading through the contibution documenation here https://nornir.readthedocs.io/en/latest/contributing/index.html#starting-development-environment and it mentions a developer environment as well as a docker-compose dependency handling.  

It states:
> Some tests requires additional services to be running which are managed by docker-compose. You can start these services with:
>make start_dev_env

Seems like these were removed in #562 but the documentation wasn't reflected.

**Comment by dbarrosop** [comment]
Yes, that needs updating. Since we separated core from plugins into different repositories there is no need for docker containers to test the core of nornir

**Comment by maroofsangi** [comment]
Hi, I was looking at this as a potential first contribution and did some digging.

The root `CONTRIBUTING.rst` (which `docs/contributing/index.rst` includes via `.. include:: ../../CONTRIBUTING.rst`) no longer mentions `make start_dev_env` or docker-compose anywhere in its current content. It looks like this was cleaned up at some point after this issue was filed, just never linked back here.

Could this be closed, or is there another doc location still referencing it that I'm missing?

**Comment by dgarros** [comment]
Thks @maroofsangi indeed it has been fixed already
